addons/mail/static/src/core/common/mail_popout_service.js

```diff
@@ -18,16 +18,21 @@ export const mailPopoutService = {
          * - Reset the external window header from main window (for appropriate title and other meta data)
          * - clear the external window's document body
          * - destroy the current app mounted on the window
+         * @param {any} id - The ID of the popout instance to reset
+         * @param {Boolean} [useDefaultAssets] - whether or not we copy the window assets to the new window.
+         * window is used.
          */
-        function reset(id) {
+        function reset(id, useDefaultAssets = true) {
```

Comment:
I'd prefer options provided as object, so the options are named.
Positional argument as boolean is not great to understand what this means, and also scalability of adding more options is poor when it's not object.